neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/model/Model.kt

```diff
@@ -38,3 +38,10 @@ data class Segment(override val id: String) : HasId {
 }
 
 data class Offer(override val id: String) : HasId
+
+data class ExCustomer(
+        override val id:String,
+        val terminationDate: String) : HasId {
```

Comment:
I would use a timestamp

Reply:
As mentioned in the comment before, but do not have a strong opinion on it.
```
As of now, this is a manual process. So, did not want to have an additional step of converting epoch long to date. Also, we have no use of time part.
```